### PR TITLE
enable type checks for <amp-accrodion>

### PR DESCRIPTION
**Pull request by sriramkrish85**

### CODE REVIEW

File: extensions/amp-accordion/0.1/amp-accordion.js [hunk 1]
```diff
@@ -21,21 +21,31 @@ import {removeFragment} from '../../../src/url';
...
+  constructor(element) {
+    super(element);
+
+    /** @private {?NodeList} */
+    this.sections_ = null;
+
+    /** @private {?string} */
+    this.id_ = null;
+
+    /** @private {?Object|undefined} */
```
Comment (cramforce) [inline extensions/amp-accordion/0.1/amp-accordion.js]: Can this be changed to either not accept null or undefined? If no, please document what they mean.

Comment (sriramkrish85) [inline extensions/amp-accordion/0.1/amp-accordion.js]: Done

File: extensions/amp-accordion/0.1/amp-accordion.js [hunk 2]
```diff
@@ -21,21 +21,31 @@ import {removeFragment} from '../../../src/url';
...
 
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private {?NodeList} */
+    this.sections_ = null;
+
+    /** @private {?string} */
+    this.id_ = null;
```
Comment (erwinmombay) [inline extensions/amp-accordion/0.1/amp-accordion.js]: do we actually need null or can it just be falsey (empty string) initial state?

Comment (erwinmombay) [inline extensions/amp-accordion/0.1/amp-accordion.js]: nvm, you are doing assertString on it, probably better to leave as nullable then

**Pull request merged**